TVShows/TVShows/Modules/ShowDetails/Cells/ShowDetailsEpisodeCell.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  ShowDetailsEpisodeCell.swift
+//  TVShows
+//
+//  Created by Ivan Milicevic on 22/10/2018.
+//  Copyright © 2018 Ivan Milicevic. All rights reserved.
+//
+
+import UIKit
+
+class ShowDetailsEpisodeCell: UITableViewCell {
+
+    @IBOutlet weak var episodeTitleLabel: UILabel!
+    @IBOutlet weak var seasonEpisodeLabel: UILabel!
+
+    // MARK: - View Lifecycle
+    override func prepareForReuse() {
```

Comment:
If you reconfigure same property each time then you don't need to clean it in prepareForReuse. In this particular case you don't need prepare for reuse :)